ocaml/asmcomp/cmmgen.ml

```diff
@@ -1022,17 +1022,14 @@ and transl_prim_1 env p arg dbg =
   | Pget_header m ->
       box_int dbg Pnativeint m (get_header (transl env arg) dbg)
   | Pperform ->
-      Misc.fatal_error "Effects-related primitives not yet supported"
-      (* CR mshinwell: use [Runtimetags] once available
       let cont =
-        make_alloc dbg cont_tag [int_const dbg 0] ~mode:Lambda.alloc_heap
+        make_alloc dbg Runtimetags.cont_tag [int_const dbg 0]
+          ~mode:Lambda.alloc_heap
       in
-      (* CR mshinwell: Rc_normal may be wrong, but this code is unlikely
-         to be in production by then *)
+      (* CR mshinwell: check Rc_normal *)
```

Comment:
`Rc_normal` is fine here, although I would mildly prefer `Rc_nontail`. (`Rc_normal` means "allow tailcalls". Preventing them here improves backtraces of paused fibers)

Reply:
Thanks, have fixed the code as per these four comments.